engine/dlib/src/zip/README.md

```diff
@@ -1,3 +1,5 @@
 To update this folder:
 * go to https://github.com/kuba--/zip
 * Download miniz.h, zip.h and zip.c
+* Rename zip.c to zip.cpp
+* In miniz.h, remove the `extern "C"`, so that the functions will be private within zip.cpp
```

Comment:
Make sure we have the instructions.
I don't think a patch will work, and the removal of the extern is easy enough.